server/internal/graphql/resolver/operator.go

```diff
@@ -3,10 +3,13 @@ package resolver
 import (
 	"context"
 
+	"github.com/gofrs/uuid"
```

Comment:
@MUzairS15 will you add a description at the top of this file as to what it's purpose is?